app/components/Layout/Header.tsx

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import {Row, Col, Form, Button} from 'react-bootstrap';
 import Link from 'next/link';
 import LoginButton from 'components/LoginButton';
+import getConfig from 'next/config';
+
+const {NAMESPACE, HOST} = getConfig()?.publicRuntimeConfig;
+const registrationUrl = `https://sso${NAMESPACE}.pathfinder.gov.bc.ca/auth/realms/pisrwwhx/protocol/openid-connect/registrations?client_id=cas-ciip-portal&response_type=code&scope=openid&redirect_uri=${HOST}/login?auth_callback=1`;
```

Comment:
Can we do a redirect on the server-side on a GET request to `/register` instead of having this registrationUrl defined here? 
The components of the keycloak registration URL are available on the server in the `kcConfig` variable (`auth-server-url`, `realm`, `resource`)